src/services/ContactUsService.ts

```diff
@@ -0,0 +1,20 @@
+import { ContactUsDto } from "types/contactUs"
+
+import { apiService } from "./ApiService"
+
+const getContactUsEndpoint = (siteName: string): string => {
+  return `/sites/${siteName}/contactUs`
+}
+
+export const getContactUs = async (siteName: string): Promise<ContactUsDto> => {
+  const endpoint = getContactUsEndpoint(siteName)
+  return apiService.get(endpoint).then((res) => res.data)
```

Comment:
just an fyi that for the http methods, we need to hint the return type otherwise it's inferred as `any`:

```suggestion
  return apiService.get<ContactUsDto>(endpoint).then((res) => res.data)
```
```ts
return api
```

Reply:
unresolving for visibility that if we wanna do this, please do so for the other services as well! no need to do here but please raise another PR for the 3 new services introduced